src/methods/geom_relations/geom_geom_processors.jl

```diff
@@ -315,11 +338,25 @@ If out_require is true, the first polygon must have at least one interior point
 If the point is in an "allowed" location and meets all requirments, return true.
 Else, return false.
 =#
-function _polygon_polygon_process(
+@inline function _polygon_polygon_process(poly1, poly2; kw...)
+    skip, returnval = _maybe_skip_disjoint_extents(poly1, poly2; kw...)
+    if skip 
+        return returnval
+    else
+        return _polygon_polygon_process(poly1, poly2; kw...)
+    end
+end
+
+function _inner_polygon_polygon_process(
     poly1, poly2;
     in_allow, on_allow, out_allow,
     in_require, on_require, out_require,
 )
+    skip, returnval = _maybe_skip_disjoint_extents(poly1, poly2;
```

Comment:
shouldn't this be taken care of in the `_polygon_polygon_process` call? 

Also I personally like the `skip && return returnval` format. Maybe use above for the other checks?

Reply:
Yeah, these are just just copy paste errors. 

(I noticed that these functions are big enough that  calling them takes a bit of time on its own, so inlining the check helps a lot - but I messed up the boilerplate)